### ISSUE TITLE
Can tab through items but can't select with keyboard

### ISSUE THREAD

**Issue by johndubya** [space10-community/conversational-form#38]
I accessed the [demo](https://space10-community.github.io/conversational-form/) on my Windows 10 desktop using Chrome, and when I got to the "What do you do" question where you can select from a list of items, I did the following:

1. Press tab once to highlight the first option "Developer."
2. Press Enter to select the option.

The first time I pressed Enter, it moved focus to the input box below without selecting the option. I then pressed tab once to get back to the "Developer" option and hit Enter again. This time, it moved focus to the "Curious mind" option. The third time, I clicked in the input box, then tabbed to the "Developer" option and pressed Enter, and this time, it selected the "Curious mind" option and continued to the next form question.

**Expected Result**
I expect when I press enter while an option is highlighted/selected that it would select that option.

**Comment by felixnielsen** [comment]
@johndubya I just tested this, and it seems to work fine for me.
Can you do a jsbin/jsfiddle where you experience this?

**Comment by johndubya** [comment]
@felixnielsen, interesting. I'm experiencing it on the demo page linked above, so feel free to use that. I am using Chrome 54.0.2840.99 m (64-bit) on Windows 10. I ran the demo again just now, and when I got to the part asking my job, I tabbed to "Developer" again and pressed Enter, and it did the same thing: focused the input box without selecting the "Developer" item. So the behavior is the same for me.
